libvast/src/format/single_layout_reader.cpp

```diff
@@ -42,8 +41,7 @@ caf::error single_layout_reader::finish(consumer& f, caf::error result) {
 }
 
 bool single_layout_reader::reset_builder(type layout) {
-  VAST_TRACE_SCOPE("{} {}", VAST_ARG(table_slice_type_), VAST_ARG(layout));
-  builder_ = factory<table_slice_builder>::make(table_slice_type_, layout);
+  builder_ = std::make_shared<table_slice_builder>(std::move(layout));
```

Comment:
`s/layout/schema/` would also help here, might be worthwhile to do that globally where possible.

Reply:
I'll try to make some adjustments because this has been bothering me as well, but please note that most of this PR was created semi-automatically, so this will probably be more effort than this PR was so far.